services/src/main/java/org/keycloak/utils/ReservedCharValidator.java

```diff
@@ -27,7 +27,10 @@
     protected static final Logger logger = Logger.getLogger(ReservedCharValidator.class);
     
     // https://tools.ietf.org/html/rfc3986#section-2.2
-    private static final String[] RESERVED_CHARS = { ":", "/", "?", "#", "[", "@", "!", "$", 
+/*    private static final String[] RESERVED_CHARS = { ":", "/", "?", "#", "[", "@", "!", "$", 
+                                                   "&", "(", ")", "*", "+", ",", ";", "=", 
+                                                   "]", "[", "\\" };*/
```

Comment:
Did you mean to leave the commented block? Same question for `keyPressed` logic below.

Reply:
I did leave it there on purpose, just not sure if we need to or not